app/src/main/java/org/mozilla/focus/toolbar/BrowserAppBarViewModel.kt

```diff
@@ -20,7 +20,9 @@ class BrowserAppBarViewModel(
     sessionRepo: SessionRepo
 ) : ViewModel() {
 
-    private val isNavigationOverlayVisible = MutableLiveData<Boolean>()
+    private val isNavigationOverlayVisible = MutableLiveData<Boolean>().apply {
+        value = false
```

Comment:
nit: We should add a comment to explain why this is false by default. The two important points are:
- this value changes in response to the navigation overlay changing visibility
- The navigation overlay is hidden by default, particularly in the case of session restore